packages/eslint-config-react-native-community/index.js

```diff
@@ -30,13 +31,39 @@ module.exports = {
     },
   },
 
-  overrides: {
-    files: ['**/__tests__/**/*.js', '**/*.spec.js', '**/*.test.js'],
-    env: {
-      jest: true,
-      'jest/globals': true,
+  overrides: [
+    {
+      files: ['*.js'],
```

Comment:
```suggestion
      files: ['*.jsx?'],
```

Maybe that would be better?

Reply:
Not sure about it coz the react or react native codebase dosen't use jsx extension or recommend it.